src/components/modules/FileDetailsModule/index.js

```diff
@@ -50,6 +50,7 @@ const FileDetailsModule = (props) => {
         }
       }
     } else if (props.file !== file) {
+      //TODO there appears to be a bug here where the file is not being set correctly and downloads the wrong file
```

Comment:
Is this bug still present? If not we should remove this comment.

Reply:
True, will  remove in next commit